### PR TITLE
MUMUP-1987& 1979 Homepage view

### PR DESCRIPTION
**Pull request by jiayinjx**
update Homepage:
1. left sidebar: change star-icon and "Apps" to search-icon and "Search"
2. Change to "Home" instead of "MyUW Home"
3. "+ Add more to home" instead of "My Favorites" and plus button
4. " inside the page: " Add more to home" instead of "Add to Favorites"

### CODE REVIEW

File: angularjs-portal-home/src/main/webapp/my-app/layout/partials/home-header.html [hunk 1]
```diff
@@ -19,20 +19,11 @@ <h3 class="home-section-header">Notifications</h3>
...
     <li class="hidden-xs">
-      <a href='apps'
-       aria-label="link to Browse for more apps"
-       tooltip="Browse for more apps"
-       tooltip-trigger="mouseenter"
-       tooltip-placement="top"
-       tooltip-popup-delay="500">
-      <i class='fa fa-plus'></i>
-     </a>
+         <button class="btn btn-primary" onclick="window.location.href='apps'">
```
Comment (jiayinjx) [inline angularjs-portal-home/src/main/webapp/my-app/layout/partials/home-header.html]: MUMUP-1987: Change the "My Favorites" section label

File: angularjs-portal-frame/src/main/webapp/samples/sidebar.json [hunk 2]
```diff
@@ -1,7 +1,7 @@
 {"sidebar" :  
             [
                 {"title": "Home", "hyperlink" : "/web", "target" : "", "faIcon" : "fa-home", "displayFlag" : ""},
-                {"title": "Apps", "hyperlink" : "/web/apps", "target" : "", "faIcon" : "fa-star", "displayFlag" : ""},
+                {"title": "Search", "hyperlink" : "/web/apps", "target" : "", "faIcon" : "fa-search", "displayFlag" : ""},
```
Comment (jiayinjx) [inline angularjs-portal-frame/src/main/webapp/samples/sidebar.json]: MUMUP-1979: Update the menu item that refers to apps

**Comment by timlevett** [comment]
Closing as a duplicate of #267

File: angularjs-portal-home/src/main/webapp/my-app/layout/partials/marketplace-light.html [hunk 3]
```diff
@@ -4,7 +4,7 @@
        <i class="fa fa-plus"></i>
      </div>
      <div class="list-item-container">
-       <h4>Add to Favorites</h4>
+       <h4>Add more to home</h4>
```
Comment (jiayinjx) [inline angularjs-portal-home/src/main/webapp/my-app/layout/partials/marketplace-light.html]: @timlevett this might not include in #267

Comment (timlevett) [inline angularjs-portal-home/src/main/webapp/my-app/layout/partials/marketplace-light.html]: It was in one of the other ones looking at predev

**Pull request closed without merging**